MODERNIZATION_GUIDE.md

```
@@ -0,0 +1,2460 @@
+```typescript
+import type { NextConfig } from 'next';
+
+const nextConfig: NextConfig = {
+  // Webpack config...
+};
+
+export default nextConfig;
+```
+
+**After (Turbopack native):**
+```typescript
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)